index.html

```diff
@@ -194,5 +194,31 @@ <h1>About me</h1>
       </div>
     </div>
   </section>
+
+  <footer class="footer">
+    <p>I'm always interested in hearing about new projects, 
+      so if you'd like to chat please get in touch.
+    </p>
+
+    <form action="https://formspree.io/f/xjvleokg" method="POST">
+      <input type="text" placeholder="Full Name" maxlength="30" required>
+      <input type="email" placeholder="Email Address" required>
+      <textarea name="textarea" placeholder="Enter text here" maxlength="500" required></textarea>
+      <button type="submit">Get in touch</button>
+
+      <hr id="footer-divide">
+
+      <ul class="socials">
+        <li><a href="https://github.com/emmiiorji"><i class="fa-brands fa-github"></i></a></li>
+        <li><a href="https://www.linkedin.com/in/emmanuel-orji-2a8317121"><i class="fa-brands fa-linkedin-in"></i></a></li>
+        <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
+        <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+        <li><a href="#"><i class="fa-brands fa-medium"></i></a></li>
+      </ul>       
```

Comment:
- [ ] Please use the same icons from Figma design you can simply click on the icon and download it then 
Figma desing : 
![image](https://user-images.githubusercontent.com/74744787/160810138-a75d455f-096f-40fd-a76a-1f199d9877ab.png)

your desing : 
![image](https://user-images.githubusercontent.com/74744787/160810320-b1a21038-f73c-49ed-b223-c599f6884ab2.png)